.gitignore

```diff
@@ -1,35 +1,9 @@
-## Ignore LaTeX build products
-**/latex/*.aux
```

Comment:
These LaTex ignores were added to this repo when it was extracted from cardano-ledger. Since this repo has no LaTex, they are un-needed.